guides/keras_nlp/getting_started.py

```diff
@@ -0,0 +1,619 @@
+"""
```

Comment:
You will need to add this as the first entry in `children` here, so this populated in the guides section as a link under KerasNLP.

https://github.com/keras-team/keras-io/blob/3c5d6ec6b334ce0ab806dd874feca68fb95bfa6f/scripts/guides_master.py#L26